.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "mocha": true,
+    "node": true
+  },
+  "globals": {
+      "dashjs": true,
+      "WebKitMediaSource": true,
+      "MediaSource": true,
+      "WebKitMediaKeys": true,
+      "MSMediaKeys": true,
+      "MediaKeys": true
+  },
+  "parser": "babel-eslint",
+  "rules": {
+    "no-caller": 2,
+    "no-undef": 2,
+    "no-unused-vars": 0,
```

Comment:
i guess this is only temporary to accelerate this process of introducing eslint, right? otherwise it makes little sense to introduce a linter and then basically turning it off. 

we can start with this but should remember to fix the flaws in code soon.

Reply:
This configuration has been set to map previous jslint configuration.
If we turn on "no-unused-vars" we'll get many errors/warnings. But I agree to fix that in a next step.

Reply:
@dsilhavy i will raise an issue to fix the linter config in next step.